FirebaseMLModelDownloader/Tests/Unit/ModelDownloaderUnitTests.swift

```diff
@@ -162,7 +162,8 @@ final class ModelDownloaderUnitTests: XCTestCase {
     modelDownloader.listDownloadedModels { result in
       completionExpectation.fulfill()
       switch result {
-      case .success:
+      case let .success(models):
+        print(models)
```

Comment:
Nit: Can you please add a prefix to the models here? "Successfully listed models: \(models)" or something should be fine.

Reply:
Ah yes that `print` statement was unnecessary there since it's the unexpected case anyway, removed.